plugins/nv-filters/nvidia-videofx-filter.c

```diff
@@ -522,7 +535,10 @@ static bool init_blur_images(struct nvvfx_data *filter)
 
 	/* 3. Set input & output images for nv blur FX */
 	NvVFX_SetImage(filter->handle_blur, NVVFX_INPUT_IMAGE, filter->BGR_src_img);
-	NvVFX_SetImage(filter->handle_blur, NVVFX_INPUT_IMAGE_1, filter->A_dst_img);
+	if (id != S_FX_BLUR)
+		NvVFX_SetImage(filter->handle_blur, NVVFX_INPUT_IMAGE_1, filter->A_dst_img);
+	else
+		NvVFX_SetImage(filter->handle_blur, NVVFX_INPUT_IMAGE_1, NULL);
```

Comment:
Leave a blank line after unenclosed `if` statements, or enclose the `if` with curly braces.

Reply:
done